rbf/cursor_test.go

```diff
@@ -1060,7 +1062,7 @@ func TestCursor_RemoveCells(t *testing.T) {
 	//f, err := os.OpenFile("before.dot", os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 066)
 }
 
-//These aren't test i'm just using to generate graphs to look at structure
+// These aren't test i'm just using to generate graphs to look at structure
```

Comment:
This comment makes me think perhaps this function shouldn't be part of our test suite.

Reply:
I'd agree with you, but also want to keep the scope of this work to linting issues.
@kcrodgers24 is looking at our tests.
There's a lot to do, so want to focus it on completing different layers.